content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,332 @@
+(i.e. the `--max-requests-inflight` and `--max-mutating-requests-inflight`
+command-line flags) to limit the amount of outstanding work that will be
+accepted, preventing a flood of inbound requests from overloading and
+potentially crashing the API server, but these flags are not enough to ensure
+that the most important requests get through in a period of high traffic.
+
+The API Priority and Fairness feature provides a configurable mechanism to
+ensure that in a high-load situation, forward progress will be made on the right
+requests. It also allows for a limited amount of queuing, so that in the event
+of _bursty_ traffic that remains below acceptable levels on average, no requests
+need be dropped, and uses a fair queuing algorithm to prevent one stream of
+requests (for example, a poorly-behaved {{< glossary_tooltip
+text="controller" term_id="controller" >}}) from starving others, even
+at the same priority level.
+
+{{< caution >}}
+In the present implementation, requests classified as "long-running" — primarily
+watches — are not subject to the API Priority and Fairness filter. This is also
+true for the `--max-requests-inflight` flag without the API Priority and
+Fairness feature enabled.
+{{< /caution >}}
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Enabling API Priority and Fairness
+
+{{< feature-state state="alpha"  for_k8s_version="v1.18" >}}
+
+The API Priority and Fairness
+[feature](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md)
+is not enabled by default. To enable it, add the following command-line flags to
+your `kube-apiserver` invocation:
+
+```shell
+kube-apiserver \
+--feature-gates=APIPriorityAndFairness=true \
+--runtime-config=flowcontrol.apiserver.k8s.io/v1alpha1=true \
+ # …and other flags as usual
+```
+
+The command-line flag `--enable-priority-and-fairness=false` will disable the
+API Priority and Fairness feature, even if other flags have enabled it.
+
+## Concepts
+There are several distinct features involved in the API Priority and Fairness
+feature. Incoming requests are classified by attributes of the request using
+_FlowSchemas_, and assigned to priority levels. Priority levels add a degree of
+isolation by maintaining separate concurrency limits, so that requests assigned
+to different priority levels cannot starve each other. Within a priority level,
+a fair-queuing algorithm prevents requests from different _flows_ from starving
+each other, and allows for requests to be queued to prevent bursty traffic from
+causing failed requests when the average load is acceptably low.
+
+### Priority Levels
+Without the API Priority and Fairness feature enabled, overall concurrency in
+the API server is limited by the `kube-apiserver` flags
+`--max-requests-inflight` and `--max-mutating-requests-inflight`. With the
+feature enbaled, the concurrency limit defined by these flags is divided up
```

Comment:
I would be a little more explicit about the fact that the values from those flags are first _summed_ and it is the sum that is then divided up.